packages/core/src/components/Form.tsx

```diff
@@ -529,18 +532,15 @@ export default class Form<
       errorSchema = merged.errorSchema;
       errors = merged.errors;
     }
-    const idSchema = schemaUtils.toIdSchema(
-      _retrievedSchema,
-      uiSchema['ui:rootFieldId'],
-      formData,
-      props.idPrefix,
-      props.idSeparator,
-    );
+    const fieldPathId = toFieldPathId('', {
+      idPrefix: props.idPrefix || DEFAULT_ID_PREFIX,
+      idSeparator: props.idSeparator || DEFAULT_ID_SEPARATOR,
```

Comment:
Are we able to call `this.getDefaultRegistry().globalFormOptions` here to get the prefix/separator?